src/providers/recraft-ai/imageGenerate.ts

```diff
@@ -0,0 +1,84 @@
+import { RECRAFTAI } from '../../globals';
+import { ErrorResponse, ImageGenerateResponse, ProviderConfig } from '../types';
+
+export const RecraftAIImageGenerateConfig: ProviderConfig = {
+  prompt: {
+    param: 'prompt',
+    required: true,
+  },
+  style: {
+    param: 'style',
+    default: 'digital_illustration',
+  },
+  width: {
```

Comment:
width and height are not accepted by Recraft APIs. 
https://www.recraft.ai/docs#generate-image
<img width="569" alt="Screenshot 2024-12-20 at 01 08 19" src="https://github.com/user-attachments/assets/7c66222a-e6ae-4481-8dcd-9113d8a7440e" />

Reply:
Following other provider's format in code for image generation, should I keep it immutable for standardisation?

Reply:
Gateway follows OpenAI standard. So if any parameter can be mapped to OpenAI compatible parameter, it will accept it and then map it to provider specific parameter. You can check the existing. You can check other providers like segmind, lemonfox-ai, etc. which are present in the providers directory.

Reply:
Moreover, from Recraft docs, it looks like they are already OpenAI compatible. So you should be able to directly map all the fields

Reply:
Got it. Will makes the changes. Thanks